corio.py

```diff
@@ -385,6 +386,7 @@ def main(options):
         terminated_tp = None
         test_ids = list()
         while True:
+            cpu_memory_details()
```

Comment:
CPU and memory monitoring can be done as part of different process. 
Logging of cpu and memory data should not be logged in corio log file. different file should be maintained,

Reply:
This is not monitoring, we need to have logging once CPU and Memory usages reaches at some level.

Reply:
We should be adding monitoring of resources on client also.